src/modules/compliance/src/lib/Engine.cpp

```diff
@@ -21,15 +21,16 @@
 
 namespace compliance
 {
-static constexpr const char* cModuleInfo = "{\"Name\": \"Compliance\","
-                                           "\"Description\": \"Provides functionality to audit and remediate Security Baseline policies on device\","
-                                           "\"Manufacturer\": \"Microsoft\","
-                                           "\"VersionMajor\": 2,"
-                                           "\"VersionMinor\": 0,"
-                                           "\"VersionInfo\": \"Dilithium\","
-                                           "\"Components\": [\"Compliance\"],"
-                                           "\"Lifetime\": 2,"
-                                           "\"UserAccount\": 0}";
+static constexpr const char* cModuleInfo =
+    "{\"Name\": \"Compliance\","
+    "\"Description\": \"Provides functionality to audit and remediate Security Baseline policies on device\","
+    "\"Manufacturer\": \"Microsoft\","
+    "\"VersionMajor\": 2,"
+    "\"VersionMinor\": 0,"
+    "\"VersionInfo\": \"Dilithium\","
```

Comment:
>Dilithium [](http://example.com/codeflow?start=24&length=9)

This would be Selenium or Bromine. Dilithium is well past.

Reply:
Development started in Dilithium, not sure what the purpose of this field is.

Dropped the value.